README.md

```diff
@@ -8,10 +8,24 @@ I have the time in milliseconds and I want it to become "30 minutes" or "3 days,
 
 **This library is actively maintained but no new features will be added.**
 
+
+## Installation
+
+This package is available as *humanize-duration* on [npm](https://www.npmjs.com/package/humanize-duration) and Bower. You can also include the JavaScript file in the browser.
+
+```bash
+npm install humanize-duration
+```
+
+Or using `yarn`
```

Comment:
I'm personally opposed to Yarn (long story) and wouldn't document it in my readme. Could you remove?

Reply:
No problem bro, I will remove